### ISSUE TITLE
Broken JSON checks

### ISSUE THREAD

**Issue by chillum** [flycheck/flycheck#645]
Hi,

For several months I have Flycheck and jsonlint installed. From some recent update, Flycheck does not work with JSON files, displaying "FlyC-" and not running the checker. It used to work before. YAML checking works for me still, but the JSON got lost.

I have jsonlint:
```sh
$ jsonlint -v
1.6.2
```

I have `flycheck-checkers` set to the default value (I can verify it).

What else information should I report to have this issue described?

**Comment by lunaryorn** [comment]
I'd like to see the output of `C-c ! C-c josn-jsonlint`.

**Comment by chillum** [comment]
`user-error: Cannot use syntax checker json-jsonlint in this buffer`

I'm using json-mode and have used it before, when it worked.

**Comment by lunaryorn** [comment]
The output of `C-c ! v` then, please.

**Comment by lunaryorn** [comment]
@chillum I'm pretty sure that this is an issue in your configuration or system, by the way.  I don't remember any changes to the jsonlint checker in Flycheck recently, and jsonlint itself was last updated about a year ago.

**Comment by chillum** [comment]
Hm, I found the problem: after update of Emacs for OS X to 24.5 something weird happens with `exec-path`.

Thanks for your help, this problem is really not related to Flycheck.

**Comment by chillum** [comment]
As it appears now, this problem is related to the fact I updated OS X to 10.10 and launchd.conf is not anymore supported (I used it to set my PATH).

**Comment by lunaryorn** [comment]
I would recommend to set `$PATH` in your shell configuration and then use the exec-path-from-shell package to copy it to your Emacs.
